src/Atmos/Model/lsforcing.jl

```diff
@@ -0,0 +1,81 @@
+export LSForcingModel, NoLSForcing, HadGEMVertical
+
+abstract type LSForcingModel end
+
+vars_state(::LSForcingModel, ::Prognostic, FT) = @vars()
+vars_state(::LSForcingModel, ::Auxiliary, FT) = @vars()
+vars_state(::LSForcingModel, ::Gradient, FT) = @vars()
+vars_state(::LSForcingModel, ::GradientFlux, FT) = @vars()
+vars_state(::LSForcingModel, ::UpwardIntegrals, FT) = @vars()
+vars_state(::LSForcingModel, ::DownwardIntegrals, FT) = @vars()
+
```

Comment:
```suggestion
```

Reply:
If I remove these lines the tests will fail with the error message "no method matching vars_state(::NoLSForcing, ::ClimateMachine.BalanceLaws.Auxiliary, ::Type{Int64})".

Reply:
Ah, you need to make `LSForcingModel` a subtype of `BalanceLaw`

Reply:
🤔  I think this is right for a sub-component? But maybe [do what `MoistureModel` does](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Atmos/Model/moisture.jl#L6) instead.